### PR TITLE
New martyrs

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: new/PRS13665Alaniqos.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">አላኒቆስ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAlāniqos</persName>
+                    <faith type="Christianity Orthodox Coptic"/>
+                    <floruit >Egyptian bishop</floruit>
+                    <death>He is commemorated on <date type="diesNatalis" ref="ethiocal:Tahsas27">27 Tāḫśāś</date></death>
+                </person>
+                <listRelation>
+                    <relation name="ecrm:P129i_is_subject_of" active="PRS13665Alaniqos" passive="LIT6529GadlaAbsadiAlaniqos"/>
```
Comment (MarcinKrawczuk) [inline new/PRS13665Alaniqos.xml:61]: Maybe add the reference to https://www.jstor.org/stable/24232567?

File: new/PRS13666Absadi.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">አብሳዲ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">’Absādī</persName>
+                    <faith type="Christianity Orthodox Coptic"/>
+                    <floruit >Egyptian bishop</floruit>
```
Comment (MarcinKrawczuk) [inline new/PRS13666Absadi.xml:57]: I guess we could encode the same with {nationality} and {occupation}

**Review by MarcinKrawczuk — APPROVED** [review]
I guess all these guys could have https://journals.sub.uni-hamburg.de/aethiopica/article/view/328 as bibliographic reference

File: new/PRS13665Alaniqos.xml (hunk 1 above)
Comment (eu-genia) [reply new/PRS13665Alaniqos.xml:61]: This is a reference to LIT4049GadlaAl and PRS1640Alaniqos, as far as I see it a different (local) saint

Comment (eu-genia) [reply new/PRS13665Alaniqos.xml:61]: (there is a relation
`<relation name="saws:isDifferentTo" active="PRS13665Alaniqos" passive="PRS1640Alaniqos"></relation>
`to make this clear)

Comment (MarcinKrawczuk) [reply new/PRS13665Alaniqos.xml:61]: My bad, sorry.

**Comment by eu-genia** [comment]
> I guess all these guys could have https://journals.sub.uni-hamburg.de/aethiopica/article/view/328 as bibliographic reference

Are you sure it applies? I believe the article is about saints present (also) outside of GS; these particular guys are not mentioned as far as I know.

**Pull request merged**